tests/e2e/kserve_test.go

```diff
@@ -26,6 +28,7 @@ func kserveTestSuite(t *testing.T) {
 		ComponentTestCtx: ct,
 	}
 
+	t.Run("Validate environment", componentCtx.validateEnv)
```

Comment:
May be it should be part of setUp()? I do not see a lot of sense to use assertions there.

Reply:
you mean in the `kserveTestSuite` ?

Reply:
I do not have strong opinion. It may make sense to limit to the particular test if it does not matter for others. Allows to skip unnecessary work if the test is skipped. On the other hand it may be part of general preparation of the environment. Originally I was thinking about one global place. There is one https://github.com/opendatahub-io/opendatahub-operator/blob/main/tests/e2e/helper_test.go#L459 which is called from creation suite. With the changes it also can be reconsidered.

Reply:
it make sense, I'll have a look

Reply:
done